src/Razor/src/Microsoft.VisualStudio.RazorExtension/Microsoft.VisualStudio.RazorExtension_vp20edrb_wpftmp.csproj

```diff
@@ -0,0 +1,602 @@
+<Project>
```

Comment:
I assume this was accidentally included?

Reply:
Whoops! Thanks!